client/src/AllAssets/AllAssets.js

```diff
@@ -160,69 +266,134 @@ const AllAssets = () => {
-                {currPage > 1 ? (
-                  <button
-                    type="button"
-                    className="ml-auto mr-1 p-1 w-20 bg-secondary text-text text-center rounded-md"
-                    onClick={() => prevPageFunc()}
-                  >
-                    Prev Page
+              <div className="flex flex-row">
+                <p className="flex items-center mx-auto">
+                  <button type="button" className="ml-1 text-xl">
+                    {currPage > 1 ? (
+                      <MdKeyboardArrowLeft
+                        className="ml-auto p-1 w-8 h-8 bg-text rounded-full"
+                        onClick={() => prevPageFunc()}
+                      />
+                    ) : (
+                      <MdKeyboardArrowLeft className="ml-auto p-1 w-8 h-8 bg-text rounded-full" />
+                    )}
                   </button>
-                ) : (
-                  <span className="ml-auto mr-1 p-1 w-20 bg-secondary text-text text-center rounded-md opacity-50">
+                  <span className="mr-1 p-1 w-28 text-text text-center text-xl rounded-md">
                     Prev Page
                   </span>
-                )}
-                <span className="mx-1 p-1 text-text rounded-md ">
-                  {currPage}
-                </span>
-                {currAssets.length / (currPage * itemsPerPage) > 1 ? (
-                  <button
-                    type="button"
-                    className="ml-1 mr-auto p-1 w-20 bg-secondary text-text text-center rounded-md"
-                    onClick={() => nextPageFunc()}
-                  >
-                    Next Page
-                  </button>
-                ) : (
-                  <span className="ml-1 mr-auto p-1 w-20 bg-secondary text-text text-center rounded-md opacity-50">
+                  <span className="mx-4 w-8 h-8 text-gunmetal bg-text text-center text-xl rounded-full font-semibold">
+                    {currPage}
+                  </span>
+                  <span className="ml-1 p-1 w-28 text-text text-center text-xl rounded-md">
                     Next Page
                   </span>
-                )}
-              </span>
+                  <button type="button" className="ml-1 text-xl">
+                    {currAssets.length / (currPage * itemsPerPage) > 1 ? (
+                      <MdKeyboardArrowRight
+                        className="mr-auto p-1 w-8 h-8 bg-text rounded-full"
+                        onClick={() => nextPageFunc()}
+                      />
+                    ) : (
+                      <MdKeyboardArrowRight className="mr-auto p-1 w-8 h-8 bg-text rounded-full" />
+                    )}
+                  </button>
+                </p>
+              </div>
+              <AddAsset showModal={showModal} setShowModal={setShowModal} />
             </div>
           ) : (
             <h1 className="text-text text-center m-auto">Loading...</h1>
```

Comment:
missing the last space

Reply:
I've now added new lines at the end of files